libafl/src/mutators/mod.rs

```diff
@@ -20,8 +20,14 @@ pub use grimoire::*;
 pub mod tuneable;
 pub use tuneable::*;
 
+#[cfg(feature = "multipart_inputs")]
```

Comment:
why a feature?

Reply:
There are some features that I depend on :slightly_smiling_face:

Reply:
mmmm @domenukk maybe we can remove rand_trait as a feature and always include the dep by default, any downsides (like no_std compatibility)?